eva/executor/overwrite_executor.py

```diff
@@ -0,0 +1,186 @@
+# coding=utf-8
+# Copyright 2018-2022 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+from collections import defaultdict
+
+import cv2
+import numpy as np
+import pandas as pd
+from PIL import Image as im
+
+from eva.catalog.catalog_manager import CatalogManager
+from eva.catalog.catalog_type import TableType
+from eva.executor.abstract_executor import AbstractExecutor
+from eva.expression.function_expression import FunctionExpression
+from eva.models.storage.batch import Batch
+from eva.plan_nodes.overwrite_plan import OverwritePlan
+from eva.storage.storage_engine import StorageEngine
+
+
+class OverwriteExecutor(AbstractExecutor):
+    def __init__(self, node: OverwritePlan):
+        super().__init__(node)
+        self.catalog = CatalogManager()
+        self.table_ref = self.node.table_ref
+        self.operation = self.node.operation
+
+    def make_new_video_path(self, video_path: str):
+        dirs = video_path.split("/")
```

Comment:
Try using the standard library such as pathlib for path manipulation, instead of directly working with "/"